reframe/schemas/config.json

```diff
@@ -232,7 +232,7 @@
                                     "type": "string",
                                     "enum": [
                                         "local", "pbs", "slurm",
-                                        "squeue", "torque"
+                                        "sge", "squeue", "torque"
```

Comment:
Is there a way to update this file automatically?  I did it manually :slightly_smiling_face:

Reply:
Afaik no, we update it manually too.